modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/portlet/action/AttachmentUploadMVCActionCommand.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.object.web.internal.object.entries.portlet.action;
+
+import com.liferay.object.web.internal.object.entries.upload.AttachmentUploadFileEntryHandler;
+import com.liferay.object.web.internal.object.entries.upload.AttachmentUploadResponseHandler;
+import com.liferay.portal.kernel.portlet.bridges.mvc.BaseMVCActionCommand;
+import com.liferay.upload.UploadHandler;
+
+import javax.portlet.ActionRequest;
+import javax.portlet.ActionResponse;
+
+/**
+ * @author Carolina Barbosa
+ */
+public class AttachmentUploadMVCActionCommand extends BaseMVCActionCommand {
```

Comment:
Since this is a command, wouldn't it make more sense to say `UploadAttachment`? Meaning, "upload" being the verb and "attachment" being the object. [Reference](https://en.wikipedia.org/wiki/Subject%E2%80%93verb%E2%80%93object_word_order)

Reply:
You're right. I'll change this.